src/core/style/_palette.scss

```diff
@@ -335,7 +335,9 @@ $md-light-theme-background: (
   hover:             rgba(black, 0.04), // TODO(kara): check style with Material Design UX
   card:       white,
   dialog:     white,
-  disabled-button:   rgba(black, 0.12)
+  disabled-button:   rgba(black, 0.12),
+  slide-toggle-bar: map_get($md-grey, 500),
+  slide-toggle-thumb: map_get($md-grey, 400)
```

Comment:
Thinking about this more, if these are super specific to the slide-toggle we should probably just move them to slide-toggle.scss and use `$md-is-dark-theme` directly there. Something like
```scss
$md-slide-toggle-bar-color: if($md-is-dark-theme, ..., ...);
...
```